### DISCUSSION TITLE
Finding better titles for achievements

### DISCUSSION THREAD

**Discussion by westnordost** [streetcomplete/StreetComplete#3009] [General]
Feel free to contribute. Some current achievement names are quite boring, don't fit the picture and/or difficult to translate.

Additionally, some people may have a problem with the generic masculine used currenlty for some achievement titles. This has no priority, but if a title is found that is better in other ways as well, it could be replaced too.

https://docs.google.com/spreadsheets/d/1u5Gtgv8UxsbGDMxM-oCzLfQE7I3cOinEE6OK-mhpydY/edit?usp=sharing

**Comment by amandasaurus** [comment]
Instead of “Postman”, how about [“postie”](https://en.wiktionary.org/wiki/postie)?

It's used colloquially, is gender neutral, and kinda cute and fun.

**Reply by mnalis** [reply]
Given that the name of the achievement needs not be related to vocation (e.g. _"Treasure hunt"_ which is not about treasure hunters at all, or _"Runner"_ which I at least _never_ accomplished while running), but is welcomed to be a play of words or humorous:

I'd go with _"Going postal"_ -- it can be a reference to [Terry Pratchett's](https://en.wikipedia.org/wiki/Terry_Pratchett) excellent book/movie [Going Postal](https://en.wikipedia.org/wiki/Going_Postal), or doubling as dark humor reference to idiom ["going postal"](https://idioms.thefreedictionary.com/Going+Postal); at reader's choice.

(of course, there should be instructions for translators to have a wider freedom in choosing something appropriate related to postboxes, addresses, street names and a like)

(also has a benefit of avoiding gender-related issues as noted on POeditor and SC issues before, while not sounding strained and/or dull)

**Reply by mnalis** [reply]
Or, on the slightly ironic side (but more easily translatable), _"Return to sender"_ ?

**Reply by mnalis** [reply]
or ["Street smarts"](https://idioms.thefreedictionary.com/street+smarts) (given that much of the most useful data collected are addresses), and it helps you navigate (the life and the streets)?

**Comment by paulklie** [comment]
How about "scout" or "trailblazer' instead of "outdoors"

**Reply by mnalis** [reply]
Perhaps ["Scout's honour"](https://dictionary.cambridge.org/dictionary/english/scout-s-honour) as a slight play of words?

**Reply by paulklie** [reply]
I love it!

**Comment by paulklie** [comment]
Instead of "mobility":

- The wheel deal
- Ramping up

**Comment by paulklie** [comment]
Instead of "Kind to animals":

- Animal friend
- Animal Friendship (if you want a d&d reference)
- Plant powered
